ref-tests/tests/integration.rs

```diff
@@ -75,3 +79,53 @@ fn canister_query() {
         Ok(())
     })
 }
+
+#[ignore]
+#[test]
```

Comment:
I'd consider this adjacent to this PR, since there are already two other tests marked `#[ignore]`, but I notice that I am wondering what process to follow if I wanted to run these, since presumably `cargo test` is not it.

Reply:
Nevermind, I see the github workflow now.

Reply:
I explain it somewhere (but can definitely make it clearer) that ignored tests are run against the ic-ref. Because they require a replica, they are ignored and ran separately. I'll make it clearer in docs.

Reply:
Oh it's at the top of this file in a comment.